src/utils/filter.ts

```diff
@@ -73,19 +76,19 @@ export const makeCampRoomHitFilter = ({
 };
 
 export const filterUnreadPrivateChats = (
-  chats: WithId<PrivateChatMessage>[],
-  user: FirebaseReducer.AuthState | undefined
+  chats: readonly WithId<PrivateChatMessage>[],
+  user?: FirebaseReducer.AuthState
 ) => {
-  return (
-    chats
-      ?.filter(
-        (message) =>
-          message.from !== user?.uid &&
-          message.deleted !== true &&
-          message.type === "private" &&
-          message.ts_utc.seconds > HIDE_BEFORE &&
-          message.isRead === false
-      )
-      .sort(chatSort) ?? []
-  );
+  if (!chats) return [];
+
+  return chats
+    ?.filter(
+      (message) =>
+        message.from !== user?.uid &&
```

Comment:
Also, if we added `!user` to the guard check (as presumably `message.from !== user?.uid` should never be true when `user` is `undefined`) then we can also remove the null-safe check here (and it makes it quick to visually skim and understand the logic without having to work trough the `&&`'d stuff as it would shortcut early on `!user`